Gemfile.lock

```diff
@@ -3,7 +3,7 @@ PATH
   specs:
     fluent-plugin-kubernetes_metadata_filter (2.7.1)
       fluentd (>= 0.14.0, < 1.14)
-      kubeclient (< 5)
+      kubeclient (>= 4.0.0, < 5.0.0)
```

Comment:
Do we have any testing outside units the version bump is not causing any regression? Can we document what version of kube we know to work?

Reply:
This is the error when initializing the kubeclient when we pulled v1.0.0 instead of >= v4.0.0 using bundler:

```
2021-06-10 00:50:53 +0000 [debug]: [filter_kube_metadata] Creating K8S client
/usr/local/lib/ruby/gems/2.7.0/gems/kubeclient-1.0.0/lib/kubeclient.rb:34:in `initialize': unknown keyword: :as (ArgumentError)
```

The `:as` option was introduced in the kubeclient initialization gem in [kubeclient/lib/kubeclient/common.rb](https://github.com/abonas/kubeclient/blob/e37a8029275a6032a6f9b641b3800601ff1a2a0b/lib/kubeclient/common.rb) during v3.0.0 tag. I would expect everything below <= 3.0.0 of kubeclient gem to break because of this change to initialization. I am no ruby expert though.

Reply:
I scouted the tags in versions and the initialization function has not change much from v4.0.0 to v4.9.2 so it makes me think that any version over 4.0.0 would work.